src/app/GlobalStyles.tsx

```diff
@@ -10,10 +10,23 @@ const GlobalStyle = createGlobalStyle`
   --color-background-secondary: white;
   --color-primary: white;
   --color-secondary: #F9AA33;
+
+}
+
+:-webkit-any-link {
+  text-decoration: none
+}
+
+:-moz-any-link {
+  text-decoration: none
+}
+
+:any-link {
+  text-decoration: none
```

Comment:
This should be sufficient, because styled-components adds all vendor prefixes as far as I know